src/miner_discovery_worker.erl

```diff
@@ -75,7 +75,8 @@ handle_info(
         lists:nth(rand:uniform(length(FreqList)), FreqList)
     end,
     Spreading = spreading(Region, byte_size(Packet)),
-    ok = miner_lora:send_poc(
+    %% TODO: temporarily ignore send_poc result
```

Comment:
Why?

Reply:
Artifact from some testing, removed and matched back to `ok` return